pkg/reconciler/broker/broker.go

```diff
@@ -22,55 +22,38 @@ import (
 	"context"
 	"fmt"
 
-	"cloud.google.com/go/pubsub"
 	"github.com/google/knative-gcp/pkg/logging"
-	"go.uber.org/multierr"
+	"github.com/google/knative-gcp/pkg/reconciler/celltenant"
 	"go.uber.org/zap"
 	corev1 "k8s.io/api/core/v1"
 	pkgreconciler "knative.dev/pkg/reconciler"
 
 	brokerv1beta1 "github.com/google/knative-gcp/pkg/apis/broker/v1beta1"
-	"github.com/google/knative-gcp/pkg/apis/configs/dataresidency"
 	brokerreconciler "github.com/google/knative-gcp/pkg/client/injection/reconciler/broker/v1beta1/broker"
-	inteventslisters "github.com/google/knative-gcp/pkg/client/listers/intevents/v1alpha1"
-	metadataClient "github.com/google/knative-gcp/pkg/gclient/metadata"
-	"github.com/google/knative-gcp/pkg/reconciler"
-	"github.com/google/knative-gcp/pkg/reconciler/broker/resources"
-	reconcilerutilspubsub "github.com/google/knative-gcp/pkg/reconciler/utils/pubsub"
-	"github.com/google/knative-gcp/pkg/utils"
 )
 
 const (
 	// Name of the corev1.Events emitted from the Broker reconciliation process.
-	brokerReconciled  = "BrokerReconciled"
-	brokerFinalized   = "BrokerFinalized"
-	brokerCellCreated = "BrokerCellCreated"
+	brokerReconciled = "BrokerReconciled"
+	brokerFinalized  = "BrokerFinalized"
 )
 
 type Reconciler struct {
-	*reconciler.Base
-
-	// listers index properties about resources
-	brokerCellLister inteventslisters.BrokerCellLister
-
-	projectID string
-
-	// pubsubClient is used as the Pubsub client when present.
-	pubsubClient *pubsub.Client
-
-	dataresidencyStore *dataresidency.Store
-	// clusterRegion is the region where GKE is running
-	clusterRegion string
+	celltenant.Reconciler
 }
 
 // Check that Reconciler implements Interface
 var _ brokerreconciler.Interface = (*Reconciler)(nil)
 var _ brokerreconciler.Finalizer = (*Reconciler)(nil)
 
-var brokerGVK = brokerv1beta1.SchemeGroupVersion.WithKind("Broker")
-
 func (r *Reconciler) ReconcileKind(ctx context.Context, b *brokerv1beta1.Broker) pkgreconciler.Event {
-	if err := r.reconcileBroker(ctx, b); err != nil {
+	logger := logging.FromContext(ctx)
+	logger.Debug("Reconciling Broker", zap.Any("broker", b))
+	b.Status.InitializeConditions()
+	b.Status.ObservedGeneration = b.Generation
+
+	bcs := celltenant.StatusableFromBroker(b)
+	if err := r.Reconciler.ReconcileGCPCellAddressable(ctx, bcs); err != nil {
```

Comment:
What's the meaning of `Addressable` here? Should it be sth like `ReconcileGCPCellTenant`?